pyvista/utilities/reader.py

```diff
@@ -2384,4 +2446,5 @@ class GIFReader(BaseReader):
     '.vtr': XMLRectilinearGridReader,
     '.vts': XMLStructuredGridReader,
     '.vtu': XMLUnstructuredGridReader,
+    '.xdmf': XdmfReader,
```

Comment:
We need to add into the documentation  https://github.com/pyvista/pyvista/blob/main/doc/api/readers/index.rst

Reply:
Thanks. I added at https://github.com/pyvista/pyvista/pull/2839/commits/95ce411236bee0f4ff17ec0de9c9d6ea0ed114da.